src/main/webapp/app/shared/notification/notification-popup/notification-popup.component.ts

```diff
@@ -96,6 +114,28 @@ export class NotificationPopupComponent implements OnInit {
         }
     }
 
+    /**
+     * Adds a notification about a updated exercise during a live exam to the component's state
+     * and pushes updated problemStatement to student exam exercise via BehaviorSubjects
+     *
+     * @param notification {Notification}
+     */
+    private addExamUpdateNotification(notification: Notification): void {
+        if (!notification.target) {
+            return;
+        }
+        try {
+            const target = JSON.parse(notification.target);
+            this.examExerciseUpdateService.updateLiveExamExercise(target.exercise, target.problemStatement);
+        } catch (error) {
+            throw new Error(error);
```

Comment:
Did you check if/where the error is cought? Otherwise using jhiAlertService.onError might be a better solution

Reply:
No, I did not. Thank you for this advice. I changed it.